pkg/bootstrapper/bootstrapper.go

```diff
@@ -276,6 +311,10 @@ func (wmcb *winNodeBootstrapper) createKubeletService() error {
 	if v, ok := wmcb.kubeletArgs["v"]; ok {
 		kubeletArgs = append(kubeletArgs, "--v="+v)
 	}
+	if v, ok := wmcb.kubeletArgs[cloudConfigOption]; ok {
```

Comment:
Shouldn't this be `cloudConfigOption` instead of `v`. v is for log-level in the previous block.

Reply:
I took `v` for a general variable. I cannot use `cloudConfigOption` as there is a constant with that name. I will use `cloudConfigValue`.